.github/workflows/lint_python.yml

```
@@ -0,0 +1,24 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
```

So the check does not run doubled when pushing to pull requests:
```suggestion
  lint_python:
    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.owner.login != github.event.pull_request.base.repo.owner.login
```